app/controllers/api/v1/admin/user/skills_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::Admin::User::SkillsController < Api::V1::BaseController
+  before_action :authenticate_user!
+
+  def create
+    @user_skill = UserSkill.new(user_skill_params)
+    authorize @user_skill
+    @user = User.find(params[:user_skill][:user_id])
```

Comment:
1. why do you pass `user_id` in scope of `user_skill`?
2. You should grab `user_id` from `params[:user_id]`